xaitk_saliency/impls/gen_detector_prop_sal/drise_scoring.py

```diff
@@ -39,14 +39,14 @@ def __init__(
                              "may not be available in scipy")
 
     def iou(self, box_a: np.ndarray, box_b: np.ndarray) -> np.ndarray:
-        """Compute the intersection over union (IoU) of two sets of boxes.
-        E.g.:
-            A ∩ B / A ∪ B = A ∩ B / (area(A) + area(B) - A ∩ B)
-        Args:
-            box_a: (np.array) bounding boxes, Shape: [A,4]
-            box_b: (np.array) bounding boxes, Shape: [B,4]
-        Return:
-            iou: (np.array), Shape: [A,B].
+        """
+        | Compute the intersection over union (IoU) of two sets of boxes.
+        | E.g.:
+        |    A ∩ B / A ∪ B = A ∩ B / (area(A) + area(B) - A ∩ B)
```

Comment:
We can probably dis-include the title sentence of the doc-string from the formatted block
```suggestion
        Compute the intersection over union (IoU) of two sets of boxes.

        | E.g.:
        |    A ∩ B / A ∪ B = A ∩ B / (area(A) + area(B) - A ∩ B)
```